### PR TITLE
Add driveMs, onDutyMs, pendingDriveMs, and pendingOnDutyMs to hos_logs_summary

### PR DESCRIPTION
**Pull request by faizandfurious**

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -4430,7 +4430,31 @@
                     "format": "int64",
                     "description": "The amount of driving time in violation in this cycle (in ms).",
                     "example": 50400000
-                  }
+                  },
+                  "onDutyMs": {
```
Comment (silverspace) [inline swagger.json]: Nit: this looks like a unix timestamp until you read the description. Can we call this `onDutyDurationMs`? 

Also, it's not clear whether this is the current shift, cycle, etc, unless you read the description. Maybe `currentCycle { onDutyDurationMs }`?

File: swagger.json [hunk 2]
```diff
@@ -4430,7 +4430,31 @@
...
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of on duty time in this cycle (in ms).",
```
Comment (silverspace) [inline swagger.json]: nit: s/ms/milliseconds

File: swagger.json [hunk 3]
```diff
@@ -4430,7 +4430,31 @@
...
+                  "driveMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of driving time in this cycle (in ms).",
+                    "example": 34200000
+                  },
+                  "pendingOnDutyMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of on duty time in this cycle for pending logs (in ms).",
```
Comment (silverspace) [inline swagger.json]: What is a pending log? Does the above OnDutyDurationMs value include pending logs?

Comment (faizandfurious) [reply swagger.json]: I can rephrase this. `pendingOnDutyMs` represents the amount of on duty time in this cycle, including any pending logs.

File: swagger.json (hunk 1 above)
Comment (faizandfurious) [reply swagger.json]: The naming convention here is consistent with the naming convention in `hos_daily_logs`

Comment (faizandfurious) [reply swagger.json]: But I see that there's a few different time periods represented in this endpoint. @dsharrison any opinion on naming?

Comment (dsharrison) [reply swagger.json]: Good question. I don't have a strong preference, but think we should favor consistency within our API.

**Comment by faizandfurious** [comment]
Opening a new PR for this.

**Pull request closed without merging**